composer/utils/object_store/uc_object_store.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import annotations
+
+import logging
+import os
+import pathlib
+import uuid
+from typing import Callable
+
+from composer.utils.import_helpers import MissingConditionalImportError
+from composer.utils.object_store.object_store import ObjectStore
+
+log = logging.getLogger(__name__)
+
+
+class UCObjectStore(ObjectStore):
```

Comment:
can you add `list_objects` please?

Reply:
The current sdk that we are using does not have a list_objects API. Let me try to find out if there is a way to fetch this data. Were do we use list objects in the logging workflow?

Reply:
we need to download all files from a folder (e.g. for the continued pretraining api)